### PR TITLE
Remove unnecessary null from createLoadingSelector

### PR DESCRIPTION
**Pull request by fpersico**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/store/reducers/loading.ts [hunk 1]
```diff
@@ -5,8 +5,6 @@
  * - ACTION_NAME_(REQUEST|SUCCESS|FAILURE)
  */
 
-import get from "lodash/get";
```
Comment (cloudify) [inline ts/store/reducers/loading.ts]: do we still need `lodash` ?

File: ts/store/reducers/loading.ts [hunk 2]
```diff
@@ -29,10 +27,10 @@ export const INITIAL_STATE: LoadingState = {
  */
 export const createLoadingSelector = (
   actions: ReadonlyArray<FetchRequestActionsType>
-): ((_: GlobalState) => boolean | null) => (state: GlobalState): boolean => {
+): ((_: GlobalState) => boolean) => (state: GlobalState): boolean => {
   // Returns true only when all actions are not loading
```
Comment (cloudify) [inline ts/store/reducers/loading.ts]: Returns true when there's no "loading" action going on

File: ts/store/reducers/loading.ts (hunk 1 above)
Comment (fpersico) [reply ts/store/reducers/loading.ts]: Not in this file but we still need the package for `lodash/merge` in `ts/theme/index.ts`

File: ts/store/reducers/loading.ts (hunk 2 above)
Comment (fpersico) [reply ts/store/reducers/loading.ts]: Sorry the description was wrong. I think a good description is `Returns true if any of the actions passed as parameter is in "loading" state`

Comment (cloudify) [reply ts/store/reducers/loading.ts]: yes much better

File: ts/store/reducers/loading.ts (hunk 1 above)
Comment (cloudify) [reply ts/store/reducers/loading.ts]: ok

**Pull request merged**